cypress/integration/buy-shirt.cy.ts

```diff
@@ -1,28 +1,34 @@
+import {AddressStepPage, LoginPage, MenuContentPage, PaymentStepPage,
+  ProductsListPage, ShippingStepPage, ShoppingCartPage} from "../page/index";
```

Comment:
this can be split into several lines:
```suggestion
import {
  MenuContentPage,
  ProductListPage,
  ShoppingCartPage,
  LoginPage,
  AddressStepPage,
  ShippingStepPage,
  PaymentStepPage,
} from "../page/index";
```